src/AlwaysMaskedField.tsx

```diff
@@ -128,7 +128,7 @@ class AlwaysMaskedField extends React.Component<AlwaysMaskedFieldProps, MaskedFi
   }
 
   private setSelection(start: number, end = start) {
-    if (this.input === document.activeElement) {
+    if (this.input && this.input === document.activeElement) {
```

Comment:
What was this change for?

Reply:
Some change to the types caused it to (rightly) believe that `this.input` can sometimes be `null`, so I have to check it before using it. In reality, it probably never will be, but it won't compile without this check.

Reply:
Oooh interesting, cool thanks for explaining